services/cd-service/pkg/repository/repository.go

```diff
@@ -2324,14 +2362,58 @@ func (s *State) GetApplicationRelease(application string, version uint64) (*Rele
 	return &release, nil
 }
 
-func (s *State) GetApplicationReleaseManifests(application string, version uint64) (map[string]*api.Manifest, error) {
+func (s *State) GetApplicationReleaseManifests(ctx context.Context, transaction *sql.Tx, application string, version uint64) (map[string]*api.Manifest, error) {
+	manifests := map[string]*api.Manifest{}
+	if s.DBHandler.ShouldUseOtherTables() {
+		release, err := s.DBHandler.DBSelectReleaseByVersion(ctx, transaction, application, version)
+		if err != nil {
+			return nil, fmt.Errorf("could not get release for app %s with version %v: %w", application, version, err)
+		}
+		for index, mani := range release.Manifests.Manifests {
+			manifests[index] = &api.Manifest{
+				Environment: index,
+				Content:     mani,
+			}
+		}
+		return manifests, nil
+	} else {
+		dir := manifestDirectoryWithReleasesVersion(s.Filesystem, application, version)
```

Comment:
This `else` looks identical to `GetApplicationReleaseManifestsFromFile`, is this intentional? If not, let's consolidate them.

Reply:
done